packages/core/test/oracle/Finder.js

```diff
@@ -12,15 +12,19 @@ describe("Finder", function () {
   let owner;
   let user;
 
+  let runningOptimism;
   before(async function () {
     accounts = await web3.eth.getAccounts();
     [owner, user] = accounts;
-    await runDefaultFixture(hre);
+
+    // If connected to Optimism network, then we can't compile some of the other contracts required in the default
+    // test fixture, so we'll skip them for now.
+    runningOptimism = (await web3.eth.net.getId()) === 420;
```

Comment:
@chrismaree to demonstrate this test against the `optimism` network, we need to skip the default test fixture for this special case. We can remove this once solidity 0.8 is fully supported

Reply:
does that not mean this fixture is not required within this test, in general?

Reply:
Good point. I'll remove it